data/json/items/comestibles/drink_other.json

```
@@ -341,5 +341,50 @@
     "charges": 3,
     "flags": [ "NO_AUTO_CONSUME" ],
     "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "salça",
+    "name": { "str_sp": "salça" },
+    "weight": "175 g",
+    "color": "red",
+    "spoils_in": "14 days",
+    "container": "jar_glass_sealed",
+    "sealed": false,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 2,
+    "calories": 42,
+    "description": "Tomato paste of Turkish origin made from pureed in-season tomatoes reduced over the sun or heat.",
+    "price": 200,
+    "price_postapoc": 100,
+    "material": [ "tomato", "veggy" ],
+    "primary_material": "tomato",
+    "volume": "250 ml",
+    "flags": [ "FREEZERBURN" ],
+    "charges": 4,
+    "fun": 2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pekmez",
+    "name": { "str_sp": "pekmez" },
+    "weight": "100 g",
+    "color": "magenta",
+    "container": "jar_glass_sealed",
+    "sealed": true,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 25,
+    "calories": 280,
+    "description": "A molasses-like syrup obtained after condensing juices of grape must.",
+    "price": 200,
+    "price_postapoc": 200,
+    "material": [ "tomato", "veggy" ],
```

```suggestion
    "description": "A molasses-like syrup obtained after condensing juices of grape must.",
    "price": 200,
    "price_postapoc": 200,
    "material": [ "fruit" ],
```
I'm having a big think 🤔